packages/api/src/index.js

```diff
@@ -26,7 +26,8 @@ router.all('*', envAll)
 const auth = {
   '🤲': handler => withCorsHeaders(handler),
   '🔒': handler => withCorsHeaders(withApiOrMagicToken(handler)),
-  '👮': handler => withCorsHeaders(withMagicToken(handler))
+  '👮': handler => withCorsHeaders(withMagicToken(handler)),
+  '📌': handler => withCorsHeaders(withPinningEnabledUser(handler))
```

Comment:
So this would become:

```suggestion
  '📌': handler => auth['🔒'](withPinningEnabledUser(handler))
```

Reply:
Have implemented (and used the word `authorised` everywhere instead of `enabled`)